docs/index.html

```diff
@@ -309,4 +309,4 @@ <h2 class="fw-bold start-journey-text" style="text-align: left;">Start your jour
       </div>
     </div>
   </body>
-</html>
```

Comment:
what happened here lol

Reply:
i don't remember removing the </html> thing...
i'll investigate